src/AspNet.Security.OAuth.Moodle/AspNet.Security.OAuth.Moodle.csproj

```diff
@@ -0,0 +1,13 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>net5.0</TargetFrameworks>
+    <Description>ASP.NET Core security middleware enabling Moodle authentication.</Description>
+    <Authors>HIT-ReFreSH</Authors>
+    <PackageTags>aspnetcore;authentication;moodle;oauth;security</PackageTags>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <FrameworkReference Include="Microsoft.AspNetCore.App" />
```

Comment:
Please restore the JetBrains annotations and use them, otherwise this provider is inconsistent with the others.